apps/documentation/src/routes/documentation/cli.mdx

```diff
@@ -21,6 +19,22 @@ Tuono is the CLI that provides all the needed commands to handle the full-stack
 
 To list all the available commands, run `tuono -h`.
 
+```bash
+The react/rust fullstack framework
+
+Usage: tuono <COMMAND>
+
+Commands:
+  dev    Start the development environment
+  build  Build the production assets
+  new    Scaffold a new project
+  help   Print this message or the help of the given subcommand(s)
+
+Options:
+  -h, --help     Print help
+  -V, --version  Print version
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>It seems like there are too many consecutive spaces here. (CONSECUTIVE_SPACES[1])
Suggestions: ` `
Rule: https://community.languagetool.org/rule/show/CONSECUTIVE_SPACES?lang=en-US&subId=1
Category: TYPOGRAPHY
<!-- __reviewdog__:ChAxZTU0NGQ3ZmM1YzY5Zjg0EgxMYW5ndWFnZVRvb2w= -->